rmf_site_format/src/robot.rs

```diff
@@ -36,7 +36,7 @@ impl Default for Robot {
 }
 
 pub trait RobotProperty:
```

Comment:
Does `RobotProperty` get used inside of `rmf_site_format`? If not, we could consider moving it into `rmf_site_editor` instead.

Reply:
[b290ac5](https://github.com/open-rmf/rmf_site/pull/265/commits/b290ac56c055294497d1392f4713aeedde7671b3)